packages/api/src/middleware/acl/hasFarmAccess.js

```diff
@@ -191,35 +192,36 @@ async function fromActivity(req) {
 }
 
 async function fromManagementPlan(managementPlanId) {
-  return await knex('management_plan').where('management_plan_id', managementPlanId).join('crop_variety', 'crop_variety.crop_variety_id', 'management_plan.crop_variety_id').first();
+  return knex('management_plan').where('management_plan_id', managementPlanId)
+    .join('crop_variety', 'crop_variety.crop_variety_id', 'management_plan.crop_variety_id').first();
 }
 
 async function fromCropVariety(crop_variety_id) {
-  return await knex('crop_variety').where({ crop_variety_id }).first();
+  return knex('crop_variety').where({ crop_variety_id }).first();
 }
 
 async function fromYield(yieldId) {
-  return await knex('yield').where({ yield_id: yieldId }).first();
+  return knex('yield').where({ yield_id: yieldId }).first();
 }
 
 async function fromPrice(priceId) {
-  return await knex('price').where({ price_id: priceId }).first();
+  return knex('price').where({ price_id: priceId }).first();
 }
 
 async function fromFarmExpense(farm_expense_id) {
-  return await knex('farmExpense').where({ farm_expense_id }).first();
+  return knex('farmExpense').where({ farm_expense_id }).first();
 }
 
 async function fromFarmExpenseType(expense_type_id) {
-  return await knex('farmExpenseType').where({ expense_type_id }).first();
+  return knex('farmExpenseType').where({ expense_type_id }).first();
 }
 
 async function fromSale(sale_id) {
-  return await knex('sale').where({ sale_id }).first();
+  return knex('sale').where({ sale_id }).first();
 }
 
 async function fromOrganicCertifierSurvey(survey_id) {
-  return await knex('organicCertifierSurvey').where({ survey_id }).first();
+  return knex('organicCertifierSurvey').where({ survey_id }).first();
```

Comment:
If you are removing `await` in these functions, would you also remove the function's `async` declaration? or does it stay because the function call after the `return` is a promise?

Reply:
you are right, now they can just be functions